src/components/organisms/Header/HeaderDesktop.tsx

```diff
@@ -43,20 +38,32 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }))
 
-const HeaderDesktop: FC<HeaderProps> = ({ hreflogo, items, login }) => {
+const HeaderDesktop: FC<HeaderProps> = ({
+  hreflogo, itemsStart, itemsEnd, login,
+}) => {
   const classes = useStyles()
   const Login = login
 
   return (
     <AppBar position="fixed" className={classes.root}>
       <Toolbar>
-        <NavLink to={hreflogo}>
-          <LogoNavbar />
-        </NavLink>
-        <div className={classes.itemsContainer}>
-          {
-            !!items.length
-            && items.map((navItem: HeaderItemProps) => (
+        <Grid
+          container
+        >
+          <Grid item xs={1}>
+            <NavLink to={hreflogo}>
+              <LogoNavbar />
+            </NavLink>
+          </Grid>
+          <Grid
+            item
+            xs={6}
+            container
+            justify="flex-start"
+          >
```

Comment:
I think `container` prop of `Grid` component should not be mixed with `item` prop. Items should be wrapped inside containers